api/src/main/java/org/apache/iceberg/variants/PhysicalType.java

```diff
@@ -41,6 +41,10 @@ public enum PhysicalType {
   FLOAT(LogicalType.FLOAT, Float.class),
   BINARY(LogicalType.BINARY, ByteBuffer.class),
   STRING(LogicalType.STRING, String.class),
+  TIMENTZ(LogicalType.TIME, Long.class),
```

Comment:
Can you also remove `NTZ` from this symbol? This is the one that matters because it is public.